### PR TITLE
feat: Add BigAnimal Hosted support for biganimal_cluster resource

### PR DESCRIPTION
**Pull request by serdardalgic**
Single Node cluster examples are added. 
Please review.

### CODE REVIEW

File: examples/resources/biganimal_cluster/single_node/bah_aws/resource.tf [hunk 1]
```diff
@@ -0,0 +1,85 @@
+terraform {
```
Comment (wai-wong-edb) [inline examples/resources/biganimal_cluster/single_node/bah_aws/resource.tf:1]: No principal ids?

File: examples/resources/biganimal_cluster/single_node/bah_gcp/resource.tf [hunk 2]
```diff
@@ -0,0 +1,75 @@
+terraform {
```
Comment (wai-wong-edb) [inline examples/resources/biganimal_cluster/single_node/bah_gcp/resource.tf:1]: No service account ids?

**Review by wai-wong-edb — CHANGES_REQUESTED** [review]

File: examples/resources/biganimal_cluster/single_node/bah_aws/resource.tf (hunk 1 above)
Comment (serdardalgic) [reply examples/resources/biganimal_cluster/single_node/bah_aws/resource.tf:1]: I added those values as commented out and created https://github.com/EnterpriseDB/terraform-provider-biganimal/issues/339.

File: examples/resources/biganimal_cluster/single_node/bah_gcp/resource.tf (hunk 2 above)
Comment (serdardalgic) [reply examples/resources/biganimal_cluster/single_node/bah_gcp/resource.tf:1]: I added those values as commented out and created https://github.com/EnterpriseDB/terraform-provider-biganimal/issues/339.

**Review by wai-wong-edb — APPROVED** [review]

**Pull request merged**